authlogic.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'bcrypt', '~> 3.1'
   s.add_development_dependency 'timecop', '~> 0.7'
   s.add_development_dependency 'rubocop', '~> 0.41.2'
+  s.add_development_dependency 'actionpack', '>= 5.0'
```

Comment:
I think you want `['>= 3.2', '< 5.2']` here.

Reply:
Unfortunately Rails 4.x has Parameters that inherit from Hash whereas 5.0+ doesn't so we need 5.0+ to test against to ensure it still works. I suppose we could use appraisal to test against all versions?

Reply:
I see. We should test against all versions, but we don't need `appraisal`, we already have separate gemfiles.

```yaml
# travis.yml
gemfile:
  - test/gemfiles/Gemfile.rails-4.2.x
  - test/gemfiles/Gemfile.rails-5.0.x
  - test/gemfiles/Gemfile.rails-5.1.x
```

Reply:
Ok sounds good. How do you run the tests against all the different gemfiles locally?